src/MongoDB/BulkWrite.c

```diff
@@ -614,6 +618,13 @@ static HashTable* php_phongo_bulkwrite_get_debug_info(zval* object, int* is_temp
 	ADD_ASSOC_BOOL_EX(&retval, "executed", intern->executed);
 	ADD_ASSOC_LONG_EX(&retval, "server_id", mongoc_bulk_operation_get_hint(intern->bulk));
 
+	if (!Z_ISUNDEF(intern->session)) {
+		ADD_ASSOC_ZVAL_EX(&retval, "session", &intern->session);
+		Z_ADDREF(intern->session);
```

Comment:
Do we not have any tests that execute a bulk write with an explicit session and then dumps the bulk object? If not, I think we should that as `bulkwrite-debug-002.phpt` (splitting it off since it will require crypto and a live server). Perhaps you want to create a complementary test for the cursor as well, that dumps a cursor after executing a query with an explicit session.

Reply:
There's a test to check if a cursor prints an explicit session (cursor-session-001.phpt) and omits an implicit session (cursor-session-002.phpt). I've added one for the bulk write object as well.